src/components/molecules/OcStatusIndicators/OcStatusIndicators.vue

```diff
@@ -6,20 +6,16 @@
         :id="indicator.id"
         :key="indicator.id"
         v-oc-tooltip="indicator.label"
-        class="oc-status-indicators-indicator"
+        class="oc-status-indicators-indicator oc-background-primary-gradient oc-p-xs oc-ml-xs"
         :aria-label="indicator.label"
         :aria-describedby="getIndicatorDescriptionId(indicator)"
         appearance="raw"
+        variation="inverse"
         :data-testid="indicator.id"
         :data-test-indicator-type="indicator.type"
         @click="indicator.handler(resource, indicator.target)"
       >
-        <oc-icon
-          :name="indicator.icon"
-          size="small"
-          :fill-type="indicator.fillType"
-          variation="passive"
-        />
+        <oc-icon :name="indicator.icon" size="small" fill-type="line" variation="passive" />
```

Comment:
```suggestion
        <oc-icon :name="indicator.icon" size="small" fill-type="line" />
```

Variation passive should be default and can be omitted